app/controllers/miq_ae_tools_controller.rb

```diff
@@ -395,7 +395,7 @@ def get_form_vars
       @resolve[:new][:target_class] = params[:target_class]
       klass = CustomButton.name_to_button_class(params[:target_class])
       unless klass.nil?
-        targets = klass.all
+        targets = klass
```

Comment:
Why does this method even exist?

```ruby
Rbac.filtered(params[:target_class]).select(:id, :name)
# or
Rbac.filtered(params[:target_class].safe_constantize.select(:id, :name))
```

think the second one may work a little better in some circumstances, but both work great